ui/app/utils/default_jobs/actions.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export default `job "job-with-actions" {
```

Comment:
Random thought: does ember-cli have a way to load random files into variables? I think webpack can do it, but I'm not sure if it is exposed.

If that's possible, then having these examples in their own files could help keep them organized, formatted, and accessible by random `nomad run` commands.

Reply:
Yep, looked briefly into it and there's no reason why we shouldn't be able to do it other than time (and there's maybe an argument to be made that a web UI user and a `nomad job init` CLI user might want to see different "get started" jobspecs)